src/client/user/actions/types.ts

```diff
@@ -35,143 +37,141 @@ export enum UserAction {
   SET_FILE_UPLOAD_STATE = 'SET_FILE_UPLOAD_STATE',
 }
 
-export type SetUrlUploadStateAction = {
-  type: UserAction.SET_URL_UPLOAD_STATE
-  payload: boolean
-}
+export type SetUrlUploadStateAction = ReduxPayloadAction<
+  typeof UserAction.SET_URL_UPLOAD_STATE,
```

Comment:
is this correct? you're changing the type from `UserAction.SET_URL_UPLOAD_STATE` to `typeof UserAction.SET_URL_UPLOAD_STATE`?